src/app/core/services/auth.service.ts

```diff
@@ -89,43 +119,4 @@ export class AuthService {
       },
     });
   }
-
-  private decodeToken(token: string) {
```

Comment:
Nice that we can get rid of this - I think you *can* decode the access token in the frontend, but you can't verify it properly